assets/js/picturefill.min.js

```diff
@@ -0,0 +1,5 @@
+/*! picturefill - v3.0.2 - 2016-02-12
```

Comment:
This file is not ignored because it is being used as is, not as part of the main file bundle.